src/transaction.ts

```diff
@@ -64,22 +64,21 @@ export function buildTx(
     return addToTx(psetBase64, inputs, outs, network);
   }
 
+  const pset = decodePset(psetBase64);
+  const nbInputs = pset.data.inputs.length + inputs.length;
+  const nbOutputs = pset.data.outputs.length + recipients.length + 1;
+
   // otherwise, handle the fee output
-  const fee = createFeeOutput(
-    inputs.length,
-    recipients.length + 1,
-    satsPerByte,
-    network
-  );
+  const fee = createFeeOutput(nbInputs, nbOutputs, satsPerByte, network);
 
   const changeIndexLBTC: number = changeOutputs.findIndex(
     out => out.asset === network.assetHash
   );
```

Comment:
actually this is not enough to understand if any change output has been added to the tx.. we should check that there's an output that contains exactly the change output script.. this filter would return true in case an LBTC output is present, but it could be that it's not the change at all..